docs/architecture/porting-existing-aspnet-apps/configuration-differences.md

```diff
@@ -54,7 +54,7 @@ public class TestModel : PageModel
 
 **Figure 2-2.** Accessing configuration values with `IConfiguration`.
 
-Using the options pattern, settings access is similar but is strongly typed and more specific to the setting(s) needed by the consuming class, as Figure 2-3 demonstrates.
+Using the [options pattern](/aspnet/core/fundamentals/configuration/options), settings access is similar but is strongly typed and more specific to the setting(s) needed by the consuming class, as Figure 2-3 demonstrates.
```

Comment:
```suggestion
Using the [options pattern](/dotnet/core/extensions/options), settings access is similar but is strongly typed and more specific to the setting(s) needed by the consuming class, as Figure 2-3 demonstrates.
```

Most (if not all) of the extensions / fundamentals content was intentionally moved from ASP.NET docs to .NET docs instead, as it is applicable to all .NET apps - not just ASP.NET. With that said, I think it is appropriate to use the .NET docs like for the options pattern here instead.